images/ansible-operator/Dockerfile

```diff
@@ -11,20 +11,22 @@ RUN rustc --version
 
 # Copy python dependencies (including ansible) to be installed using Pipenv
 COPY images/ansible-operator/Pipfile* ./
+
 # Instruct pip(env) not to keep a cache of installed packages,
 # to install into the global site-packages and
 # to clear the pipenv cache as well
-ENV PIP_NO_CACHE_DIR=1 \
-    PIPENV_SYSTEM=1 \
-    PIPENV_CLEAR=1
+ENV PIP_NO_CACHE_DIR=1 
+
 # Ensure fresh metadata rather than cached metadata, install system and pip python deps,
 # and remove those not needed at runtime.
 RUN set -e && yum clean all && rm -rf /var/cache/yum/* \
   && yum update -y \
-  && yum install -y libffi-devel openssl-devel python39-devel gcc python39-pip python39-setuptools \
-  && pip3 install --upgrade pip~=23.3.2 \
-  && pip3 install pipenv==2023.11.15 \
-  && pipenv install --deploy \
+  && yum install -y python3.11 \
+  && yum install -y libffi-devel openssl-devel gcc python3.11-devel python3.11-pip python3.11-setuptools \
```

Comment:
Any specific reason to change to python3.11 and not latest python3.12?

Reply:
Historically during the development, there were package conflicts, I believe it was because of cached packages when trying manually. Hence the 3.11 got stuck. Nevertheless, 3.12 seems to work fine with no issues, upgraded the version.